requirements.txt

```
@@ -0,0 +1,47 @@
+alabaster==0.7.12
+appnope==0.1.2
+Babel==2.9.0
+backcall==0.2.0
+certifi==2020.12.5
+chardet==4.0.0
+cycler==0.10.0
+decorator==5.0.7
+docutils==0.16
+idna==2.10
+imagesize==1.2.0
+ipython==7.22.0
+ipython-genutils==0.2.0
+jedi==0.18.0
```

Need to specify a couple of dependencies exactly for `spyder==5.0.0`. 

```suggestion
jedi==0.17.2
```

---

Spyder isn't a dependency of either the documentation or the main library, so it doesn't need to be accounted for in this requirements file.  This is purely about building the documentation.

---

I was following [these instructions](http://qutip.org/docs/latest/installation.html#building-your-conda-environment) on building the environment for the main library.